src/Symfony/Component/Debug/Tests/Exception/FlattenExceptionTest.php

```diff
@@ -258,6 +258,9 @@ function () {},
 
     public function testRecursionInArguments()
     {
+        if (!isset($a)) {
```

Comment:
useless. It will never be set there

Reply:
It's used for the variable to not be undeclared in the following line.

Reply:
The correct fix to help PHPStan understand the code is

~~~php
$a = null;
$a = array('foo', array(2, &$a));
~~~